lib/solargraph/pin/block.rb

```diff
@@ -3,19 +3,22 @@
 module Solargraph
   module Pin
     class Block < Closure
-      # The signature of the method that receives this block.
-      #
       # @return [Parser::AST::Node]
       attr_reader :receiver
```

Comment:
Not sure if this was the intention, but `receiver` is the :call node, not the :def node as the comment implied